What's new in ASP.NET Core 9.0 stub (#31700)
dotnet/AspNetCore.Docs: aspnetcore/release-notes/aspnetcore-9.0.md

---
title: What's new in ASP.NET Core 9.0
author: rick-anderson
description: Learn about the new features in ASP.NET Core 9.0.
ms.author: riande
ms.custom: mvc
ms.date: 2/5/2024
uid: aspnetcore-9
---
# What's new in ASP.NET Core 9.0

This article highlights the most significant changes in ASP.NET Core 9.0 with links to relevant documentation.

This article will be updated as new preview releases are made available.

<!-- New content should be added to ~/aspnetcore-9/includes/newFeatureName.md files. This will help prevent merge conflicts in this file. -->

> ## Blazor
> 

Reviewer: ```suggestion
## Blazor

This section describes new features for Blazor.

```

---
title: What's new in ASP.NET Core 9.0
author: rick-anderson
description: Learn about the new features in ASP.NET Core 9.0.
ms.author: riande
ms.custom: mvc
ms.date: 2/5/2024
uid: aspnetcore-9
---
# What's new in ASP.NET Core 9.0

This article highlights the most significant changes in ASP.NET Core 9.0 with links to relevant documentation.

This article will be updated as new preview releases are made available.

> New content should be added to ~/aspnetcore-9/includes/newFeatureName.md files. This will help prevent merge conflicts in this file.

Reviewer 2: Should this be an HTML comment to avoid display?

> New content should be added to ~/aspnetcore-9/includes/newFeatureName.md files. This will help prevent merge conflicts in this file.

Reviewer 3: ```suggestion
<!-- New content should be added to ~/aspnetcore-9/includes/newFeatureName.md files. This will help prevent merge conflicts in this file. -->
```

## Blazor

> ### Full-stack web UI

Reviewer 2: If meant for Blazor, this one can be removed.

```suggestion
```

> 

Reviewer 2: ```suggestion
```

> ## SignalR
> 

Reviewer: ```suggestion
## SignalR

This section describes new features for authentication and authorization.

```

## Minimal APIs

This section describes new features for minimal APIs.

> ## Native AOT

Reviewer: ```suggestion
```

## Authentication and authorization

> ASP.NET Core 8 adds new features to authentication and authorization.

Reviewer: ```suggestion
This section describes new features for authentication and authorization.
```

> ASP.NET Core 8 adds new features to authentication and authorization.

Reviewer 3: ```suggestion
ASP.NET Core 9 adds new features to authentication and authorization.
```

## Miscellaneous

> The following sections describe miscellaneous new features in ASP.NET Core 8.

Reviewer: ```suggestion
This section describes miscellaneous new features.
```

> The following sections describe miscellaneous new features in ASP.NET Core 8.

Reviewer 3: ```suggestion
The following sections describe miscellaneous new features in ASP.NET Core 9.
```
